### PR TITLE
Treat svg elements the same as images

### PR DESCRIPTION
**Pull request by timkelty**

### CODE REVIEW

**Comment by AntonTrollback** [comment]
:+1:

**Comment by timkelty** [comment]
Sorry about the typo!

File: test/index.html [hunk 1]
```diff
@@ -18,6 +18,10 @@
     background: yellow;
   }
 
+  .u-hidden {
```
Comment (giuseppeg) [inline test/index.html]: why this?

Comment (timkelty) [inline test/index.html]: Good question...

Comment (timkelty) [inline test/index.html]: Ah that was just to hide the svg symbol reference so it didn't have to be copied over and over.

**Comment by timkelty** [comment]
Looking back on this, I'm not sure it is such good idea. There's a lot of goofy stuff cross-browser stuff with SVG with auto-widths, especially within a table cell. I think this would just add unnecessary confusion.

Really, I think a better fix would be to remove the `.Arrange-sizeFill img`/`.Arrange-sizeFit img` altogether in place of a caveat in the README.

Or, we could have another class where you can apply these styles manually. Having placed on all `img` within these can be problematic if you have other images within.

**Comment by timkelty** [comment]
>>> Or, we could have another class where you can apply these styles manually. Having placed on all img within these can be problematic if you have other images within.

That said, these styles feel like they could be incorporated into a utility perhaps even better. Then we can just put a note in the readme that if you're using images to force your widths, you might need to use some utility classes.

```
.Arrange-sizeFill img {
  height: auto;
  max-width: 100%;
}

.Arrange-sizeFit img {
  max-width: none !important;
  width: auto !important;
}
```

**Comment by simonsmith** [comment]
Would have been good to add new pre processor at the same time, as I think this component will need some changes to pass bem-linter and stylelint. 

We can do it separate though.

**Comment by simonsmith** [comment]
We should probably merge the `gh-pages` branch too

**Comment by timkelty** [comment]
@simonsmith Yeah, it had already been bumped to `1.0`, so I didn't see the harm in pushing a minor bump before that other stuff.

**Pull request closed without merging**